sound/soc/sof/intel/pci-tgl.c

```diff
@@ -159,6 +159,62 @@ static const struct sof_dev_desc adl_desc = {
 	.ops_init = sof_tgl_ops_init,
 };
 
+static const struct sof_dev_desc rpls_desc = {
+	.machines               = snd_soc_acpi_intel_adl_machines,
+	.alt_machines           = snd_soc_acpi_intel_adl_sdw_machines,
```

Comment:
This is on purpose reusing adl. We have no differences in machine definitions yet. If there will lbe, the machines definitions should be differentiated.

Reply:
Addressing to review feedback, adding new tables.